reporthandling/results/v1/reportsummary/controlsummarymethods.go

```diff
@@ -116,11 +116,11 @@ func (controlSummary *ControlSummary) GetScore() float32 {
 }
 
 func (controlSummary *ControlSummary) GetComplianceScore() float32 {
-	// if ComplianceScore field is set return it, else return -1
+	// if ComplianceScore field is set return it, else return 0
 	if controlSummary.ComplianceScore != nil {
 		return *controlSummary.ComplianceScore
 	}
-	return -1
+	return 0
```

Comment:
@YiscahLevySilas1 Why was it written like this in the firs place?

Reply:
Returning 0% is misleading (so is 100%) as we are neither compliant nor not compliant, we didn't run anything here. We wouldn't want this to affect our overall compliance score for e.g.
I think the solution for this should come from above - if kubescape recognizes we have no resources to run on at all we should return an infomative message

Reply:
That does make sense, though the biggest irk I had was with the -1%.

Even I had thought of the reasoning behind 0 and 100, with 0 it was of the sort "recheck the resources" and with 100 it was more of "there are no resources so no resources break posture".

In hindsight, it would've been better to set it to "N/A" instead.

Though sending an informative message also makes a lot of sense, I might be wrong about this but isn't there a PR that adds this check/message?

Reply:
I agree that “N/A” is an option, as here we are setting a number the conversion could be done by the printer package in Kubescape

Reply:
@dwertent @YiscahLevySilas1 I've made the changes, which now looks like this:

![image](https://github.com/kubescape/opa-utils/assets/81813720/90f8337b-14c7-47e0-92fd-9748f349eef8)

If this satisfies our requirements I'll go ahead and open a PR to the Kubescape repo and close the PR and issue in this repo.
Thanks!

Reply:
Thanks @XDRAGON2002 this makes sense :)
[In this PR](https://github.com/kubescape/kubescape/pull/1296) we tried to prevent this output from happening in the first place